packages/odyssey-storybook/src/components/odyssey-mui/DataTable/DataTable.stories.tsx

```diff
@@ -343,6 +349,13 @@ const filterData = ({
           });
         }
 
+        if (id === "startLetter" && typeof row.name === "string") {
+          const firstLetter = row.name[0]?.toLowerCase();
+          if (value === "vowel") return "aeiou".includes(firstLetter);
+          if (value === "consonant") return !"aeiou".includes(firstLetter);
+          return true;
+        }
```

Comment:
I have no clue why this code is here.

Reply:
Oh, this is in a story. Yeah, still not seeing it just by looking at the code.

Is it possible to add comments or descriptions to stories?